LocalSettings.php

```diff
@@ -382,7 +391,8 @@ function onBeforePageDisplay( OutputPage &$out, Skin &$skin )
 	NS_ISSUE    => true,
 	NS_GUIDE    => true,
 	NS_ORG      => true,
-	NS_EVENT    => true
+	NS_EVENT    => true,
+	NS_CSN    => true
 );
 
 #=============================================== Permissions ===============================================
```

Comment:
Also consider if any permssions need to be added to the usergroups for the new NS.